eventmesh-webhook/eventmesh-webhook-admin/src/main/java/org/apache/eventmesh/webhook/admin/FileWebHookConfigOperation.java

```diff
@@ -159,8 +159,9 @@ public List<WebHookConfig> queryWebHookConfigByManufacturer(final WebHookConfig
     private WebHookConfig getWebHookConfigFromFile(final File webhookConfigFile) {
         final StringBuilder fileContent = new StringBuilder();
 
-        try (BufferedReader br = Files.newBufferedReader(Paths.get(webhookConfigFile.getAbsolutePath()),
-            StandardCharsets.UTF_8)) {
+        try (
+                BufferedReader br = Files.newBufferedReader(Paths.get(webhookConfigFile.getAbsolutePath()),
+                        StandardCharsets.UTF_8)) {
             String line;
```

Comment:
I don't think formating like this is beautiful:

![image](https://github.com/apache/eventmesh/assets/41445332/01ff3fdc-a119-4f4a-bc74-b8be7010819f)

Reply:
![image](https://github.com/apache/eventmesh/assets/41445332/a19383a0-8229-4f03-88fa-d9ed8aa54984)

Reply:
I have not found a good solution at present, which may be caused by other limitations. I expect continuous improvement through the eventmesh community in the future.

Reply:
Some extra line breaks and indentations were added here, is this due to the plugin, or is it a rules issue?